platform/linux-dpdk/include/odp/api/plat/version_types.h

```diff
@@ -0,0 +1 @@
+../../../../../linux-generic/include/odp/api/plat/version_types.h
```

Comment:
No, the structure is not what was proposed. What was proposed is the following:

- **cloud-dev** is a copy of api-next. It accumulates changes that are intended to be merged back into api-next (and master) relating to odp cloud development.

- **odp-dpdk-import** is a copy of cloud-dev. It is the "raw dump" of odp-dpdk into cloud-dev. It serves as the source material for (restructured) merges of odp-dpdk code back into cloud-dev.

**cloud-dev** is where the "new ODP" is being built that will eventually become **master**. **odp-dpdk-import** serves two purposes:

- An archive of the odp-dpdk commit history for those interested in such things
- A staging area for work on odp-dpdk as it's being restructured for merging back into **cloud-dev**

What gets merged into **cloud-dev** is those portions of odp-dpdk (sans the detailed commit history that's not relevant to this effort) that are intended to become the new **master**.